### PR TITLE
[Agent v2.17] Stac 14511 - dynatrace support for synthetic checks

### PR DESCRIPTION
**Pull request by gggina**
Confirm will be released with v462 before merging!

### CODE REVIEW

File: stackpacks/integrations/dynatrace.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ To enable the Dynatrace topology check and begin collecting topology data from D
         # relative_time : <relative_time> # default 'hour'
         # custom_device_relative_time: 1h
         # custom_device_fields: +fromRelationships,+toRelationships,+tags,+managementZones,+properties.dnsNames,+properties.ipAddress
+        # custom_device_ip: True  # By default its True
```
Comment (gggina) [inline stackpacks/integrations/dynatrace.md:80]: Add note "Requires Agent v2.17 or later"

File: stackpacks/integrations/dynatrace.md [hunk 2]
```diff
@@ -93,6 +94,7 @@ To enable the Dynatrace topology check and begin collecting topology data from D
    - **relative_time** - The relative timeframe for retrieving topology.
    - **custom_device_relative_time** - The relative timeframe for retrieving custom devices.
    - **custom_device_fields** - Which Custom Device property fields will be used.
+   - **custom_device_ip** - Use IP address for generating Custom Devices identifier.
```
Comment (gggina) [inline stackpacks/integrations/dynatrace.md:97]: Add note "Requires Agent v2.17 or later"

File: stackpacks/integrations/dynatrace.md [hunk 3]
```diff
@@ -185,6 +187,7 @@ The API endpoints used in the StackState integration are listed below:
 * `/api/v1/entity/services`
 * `/api/v1/events`
 * `/api/v2/entities`
+* `/api/v1/synthetic/monitors`
```
Comment (gggina) [inline stackpacks/integrations/dynatrace.md:190]: Add note "Requires Agent v2.17 or later"

File: stackpacks/integrations/dynatrace.md [hunk 4]
```diff
@@ -218,10 +221,10 @@ The Dynatrace integration also understands StackState [common tags](../../config
 
 The [Dynatrace topology check](#dynatrace-topology-check) retrieves the following topology data from Dynatrace:
 
-| Data | Description |
-| :--- | :--- |
-| Components | Smartscape Applications, Hosts, Processes, Process-Groups, Services and Custom Devices. |
-| Relations | Relations between the imported components are included in the component data retrieved from Dynatrace. |
+| Data | Description                                                                                                  |
+| :--- |:-------------------------------------------------------------------------------------------------------------|
+| Components | Smartscape Applications, Hosts, Processes, Process-Groups, Services, Custom Devices and Synthetic Monitors.. |
```
Comment (gggina) [inline stackpacks/integrations/dynatrace.md:226]: "Smartscape Applications, Hosts, Processes, Process-Groups, Services, Custom Devices. Synthetic Monitors are also available when running Agent v2.17 or later."

File: stackpacks/integrations/dynatrace.md (hunk 1 above)
Comment (gggina) [inline stackpacks/integrations/dynatrace.md:80]: "Default True. Set to False if there are multiple custom devices using the same IP to disable identifier generation based on IP and prevent merging of ???."

File: stackpacks/integrations/dynatrace.md (hunk 2 above)
Comment (gggina) [reply stackpacks/integrations/dynatrace.md:97]: Set to False if there are multiple custom devices using the same IP to disable identifier generation based on IP and prevent merging of ???.

Comment (gggina) [reply stackpacks/integrations/dynatrace.md:97]: Use same text in gitlab stackpack docs

File: stackpacks/integrations/dynatrace.md (hunk 1 above)
Comment (TGCapra) [reply stackpacks/integrations/dynatrace.md:80]: Set to False if there are multiple custom devices using the same IP to disable identifier generation based on IP and prevent merging of different custom device components.

**Review by TGCapra — APPROVED** [review]

**Pull request merged**